src/app/reducers/authors.handlers.ts

```diff
@@ -7,23 +7,32 @@ import { getReorderedAffiliations } from 'app/reducers/affiliations.handlers';
 import { UpdateObjectChange } from 'app/utils/history/update-object-change';
 import { AddObjectChange } from 'app/utils/history/add-object-change';
 import { DeleteObjectChange } from 'app/utils/history/delete-object-change';
+import { ArticleInformation } from 'app/models/article-information';
+import { Change } from 'app/utils/history/change';
+
+function getUpdatedArticleInfo(articleInfo: ArticleInformation, authors: Person[]): Change {
+  const newArticleInfo = articleInfo.clone();
+  newArticleInfo.updateCopyrightStatement(authors);
```

Comment:
Just tracking through the code just to understand how this is triggered as my first thought was that this should just be triggered by the `UPDATE_AUTHOR` action, but I can see that it already is and that is how the code gets here. Made me wonder if we could have multiply handlers/reducers registered for actions that are executed in order, hence a change to the state for an author triggers the author handlers, followed by the update Copyright handler, etc. Anyhow, doesn't need to be looked into now but just wanted to pen my thoughts.